multisig/proposals/vip-003/vip-003-sepolia.ts

```diff
@@ -0,0 +1,209 @@
+import { NETWORK_ADDRESSES, ZERO_ADDRESS } from "../../../src/networkAddresses";
+import { makeProposal } from "../../../src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+const REDSTONE_XVS_FEED = "0x0d7697a15bce933cE8671Ba3D60ab062dA216C60";
+const CHAINLINK_BTC_FEED = "0x1b44F3514812d835EB1BDB0acB33d3fA3351Ee43";
+const CHAINLINK_ETH_FEED = "0x694AA1769357215DE4FAC081bf1f309aDC325306";
+const CHAINLINK_USDC_FEED = "0xA2F78ab2355fe2f984D808B5CeE7FD0A93D5270E";
+
+// VIP: Configures the new oracle with the ACL and configures the initial price feeds on this oracle
+export const vip003 = () => {
+  return makeProposal([
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "pause()", sepolia.NORMAL_TIMELOCK],
+    },
+    {
+      target: sepolia.ACM,
+      signature: "giveCallPermission(address,string,address)",
+      params: [ZERO_ADDRESS, "unpause()", sepolia.NORMAL_TIMELOCK],
```

Comment:
Don't use ZERO_ADDRESs for pause/unpause. I would use only specific addresses in this VIP